CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 Change Log
 ==========
 
+Version 1.1.0 *(26th May 2023)*
+-------------------------------------------
+- Adds support for Remote Config Variables. Please refer to the [Remote Config Variables doc](/docs/Variables.md) to read more on how to integrate this to your app.
```

Comment:
@piyush-kukadiya sync with @nishant-clevertap and @shivamsharma2710 to add changelog for streamlining of callbacks